dask/bytes/tests/test_hdfs.py

```diff
@@ -297,8 +305,8 @@ def test_glob(hdfs):
             {basedir + p for p in ['/a', '/a1', '/a2', '/a3', '/b1', '/c', '/c2']})
 
 
+@pytest.mark.skipif(not distributed)
 def test_distributed(hdfs):
```

Comment:
Thanks for doing this.

I believe that this will also need the `loop` pytest fixture.